package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vizlab-home",
-  "version": "0.0.1",
+  "version": "0.12.1",
```

Comment:
will we ever see v1.0?

Reply:
Hah I was also wondering why this version numbering is the way it is. I think it would make sense to be 1. something at this point

Reply:
Please create an issue with any lingering tasks needed to publish v1.0 of this site and assign it to yourself. I would like to keep it under 10 hours and to be accomplished in Q1.